### PR TITLE
[W5.4r][W15-A1] Poh Jie

### PR DESCRIPTION
**Pull request by pohjie**
@okkhoy My deepest apologies for this late submission. I had mid-terms on Friday of Week 6 and I blocked everything out. I was not aware that this was due by that week's tutorial. Of course, this is not a valid excuse as my fellow peers got their job done- however, I will like to ask for a second chance as prior to this late submission I've gotten my work done on time and prepared for tutorials. On my part, I will be more aware of various modules' announcements and check in with my group mates week by week for necessary submissions. 

Sincerely,
Poh Jie

### CODE REVIEW

File: src/seedu/addressbook/data/person/sortByPerson.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+import java.util.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/sortByPerson.java:3]: don't import using `*` import only the classes that are necessary

File: src/seedu/addressbook/data/person/sortByPerson.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class sortByPerson implements Comparator<Person> {
+    public int compare(Person o1, Person o2) {
+        String o1Name = o1.getName().toString();
+        String o2Name = o2.getName().toString();
+
+        if (o1Name.compareTo(o2Name) < 0) {
+            return -1;
+        }
+        else if (o1Name.compareTo(o2Name) > 0) {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/sortByPerson.java:13]: coding standard violation here

**Review by okkhoy — COMMENTED** [review]
Please pay attention to coding standards.

**Comment by okkhoy** [comment]
@pohjie 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/sortByPerson.java (hunk 2 above)
Comment (pohjie) [reply src/seedu/addressbook/data/person/sortByPerson.java:13]: Hi I don't quite get why this is a coding standard violation- read through the coding style guide and I don't spot my mistake.

**Comment by pohjie** [comment]
Acknowledged with thanks.

**Pull request closed without merging**